src/current/v24.3/create-logical-replication-stream.md

```diff
@@ -0,0 +1,148 @@
+
+## LDR modes
+
+_Modes_ determine how LDR replicates the data to the destination cluster. There are two modes:
+
+- `immediate` (default): Attempts to replicate the changed row directly into the destination table, without re-running constraint validations or triggers. It does not support writing into tables with [foreign key]({% link {{ page.version.version }}/foreign-key.md %}) constraints.
+- `validated`: Attempts to apply the write in a similar way to a user-run query, which would re-run all constraint validations or triggers relevant to the destination table(s). It will potentially reject the change if it fails rather than writing it, which will cause the row change to enter the DLQ instead.
+
+## Examples
+
+To start LDR, you must run the `CREATE LOGICAL REPLICATION STREAM` statement from the **destination** cluster. Use the[ fully qualified table name(s)]({% link {{ page.version.version }}/sql-name-resolution.md %}#how-name-resolution-works) and ensure that the source and destination table names are identical. The following examples show statement usage with different options and use cases.
+
+### Start an LDR stream
+
+{% include {{ page.version.version }}/ldr/multiple-tables.md %}
+
+#### Single table
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE LOGICAL REPLICATION STREAM FROM TABLE {database.public.table_name} ON 'external://{source_external_connection}' INTO TABLE {database.public.table_name};
+~~~
+
+#### Multiple tables
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE LOGICAL REPLICATION STREAM FROM TABLES ({database.public.table_name},{database.public.table_name},...)  ON 'external://{source_external_connection}' INTO TABLES ({database.public.table_name},{database.public.table_name},...);
+~~~
+
+### Ignore row-level TTL deletes
+
+If you would like to ignore [row-level TTL]({% link {{ page.version.version }}/row-level-ttl.md %}) deletes in the LDR stream, set the [`ttl_disable_changefeed_replication` storage parameter]({% link {{ page.version.version }}/row-level-ttl.md %}#ttl-storage-parameters) on the table. On the **source** cluster, alter the table to set the table storage parameter:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+ALTER TABLE {table_name} SET (ttl_disable_changefeed_replication = 'true');
+~~~
+
+When you start LDR on the **destination cluster**, include the `discard = ttl-deletes` option in the statement:
+
+{% include_cached copy-clipboard.html %}
+~~~ sql
+CREATE LOGICAL REPLICATION STREAM FROM TABLE {database.public.table_name} ON 'external://{source_external_connection}' INTO TABLE {database.public.table_name} WITH discard = ttl-deletes;
+~~~
+
+### Bootstrap LDR from a backup
+
+To start LDR from a specific point in time without an initial backfill of the destination table, you can use a backup and the `cursor` option with `CREATE LOGICAL REPLICATION STREAM`.
```

Comment:
just a heads up, once jeff's PR lands, we'll add one more step around setting a cluster setting before running the backup.